packages/storybook/src/nimble/button/button.mdx

```diff
@@ -1,45 +1,269 @@
 import { Canvas, Meta, Controls, Title } from '@storybook/addon-docs/blocks';
+import ButtonAnatomy from './images/Button.png';
 import ContentHiddenDocs from '../patterns/button/content-hidden-docs.mdx';
 import StylingDocs from '../patterns/button/styling-docs.mdx';
 import ComponentApisLink from '../../docs/component-apis-link.mdx';
 import { buttonTag } from '@ni/nimble-components/dist/esm/button';
 import { anchorButtonTag } from '@ni/nimble-components/dist/esm/anchor-button';
 import * as buttonStories from './button.stories';
+import { NimbleIconKey } from '@ni/nimble-react/dist/esm/icons/key';
+import { NimbleButton } from '@ni/nimble-react/dist/esm/button';
+import { NimbleSelect } from '@ni/nimble-react/dist/esm/select';
+import { NimbleListOption } from '@ni/nimble-react/dist/esm/list-option';
 
 <Meta of={buttonStories} />
 <Title of={buttonStories} />
 
-Per [W3C](https://www.w3.org/WAI/ARIA/apg/patterns/button/) - A button is a
-widget that enables users to trigger an action or event, such as submitting a
-form, opening a dialog, canceling an action, or performing a delete operation.
+## Overview
 
-Consider <Tag name={anchorButtonTag} /> if you want a button that triggers
-navigation to a URL.
+A button is a widget that enables users to trigger an action or event, such as
+submitting a form, opening a dialog, canceling an action, or performing a delete
+operation. _Source: [W3C](https://www.w3.org/WAI/ARIA/apg/patterns/button/)_
 
-<Canvas of={buttonStories.outlineButton} />
+<TableOfContents
+    title="On this page"
+    headingSelector="h2"
+    exclude={['Overview']}
+/>
+
+<img
+    alt="Button anatomy"
+    src={ButtonAnatomy}
+    style={{ maxWidth: '936px', width: '100%', display: 'block' }}
+/>
 
-## API
+## Implementation & API
 
 <Controls of={buttonStories.outlineButton} />
 
 <ComponentApisLink />
 
-## Styling
+## Usage
+
+### When to Use
+
+- Primary Actions: Use buttons for primary actions that users need to take, such
+  as submitting a form, saving changes, or proceeding to the next step.
+- Call to Action (CTA): Buttons are ideal for CTAs that encourage users to take
+  a specific action, like “Sign Up,” “Buy Now,” or “Learn More.”
```

Comment:
This list seems incomplete. You use buttons for secondary and tertiary actions too (by my definition, "OK" is a primary action while "Cancel" would be a secondary action). And in toolbars and many other situations where there is no hierarchy/precedence. 

I was also expecting the list to direct me to how to configure the button for each of these use cases but "CTA" isn't used elsewhere in the document and this use of "Primary" doesn't tie back to our `appearance-variant='primary'`.

After reading a bit more would it make more sense for this to correspond to the numbered sub-headings under Appearance & Styling? Or just remove this list and let users discover the info below?